routing.rst

```diff
@@ -210,103 +166,181 @@ Symfony provides a handy way to declare localized routes without duplication.
             xsi:schemaLocation="http://symfony.com/schema/routing
                 https://symfony.com/schema/routing/routing-1.0.xsd">
 
-            <route id="about_us" controller="App\Controller\CompanyController::about">
-                <path locale="nl">/over-ons</path>
-                <path locale="en">/about-us</path>
-            </route>
+            <route id="api_post_show" path="/api/posts/{id}"
+                controller="App\Controller\BlogApiController::show"
+                methods="GET|HEAD"/>
+
+            <route id="api_post_edit" path="/api/posts/{id}"
+                controller="App\Controller\BlogApiController::edit"
+                methods="PUT"/>
         </routes>
 
     .. code-block:: php
 
         // config/routes.php
-        use App\Controller\CompanyController;
+        use App\Controller\BlogApiController;
         use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
 
         return function (RoutingConfigurator $routes) {
-            $routes->add('about_us', [
-                'nl' => '/over-ons',
-                'en' => '/about-us',
-            ])
-                ->controller([CompanyController::class, 'about'])
+            $routes->add('api_post_show', '/api/posts/{id}')
+                ->controller([BlogApiController::class, 'show'])
+                ->methods(['GET', 'HEAD'])
+            ;
+            $routes->add('api_post_edit', '/api/posts/{id}')
+                ->controller([BlogApiController::class, 'edit'])
+                ->methods(['PUT'])
             ;
         };
 
-When a localized route is matched Symfony automatically knows which locale
-should be used during the request. Defining routes this way also eliminated the
-need for duplicate registration of routes which minimizes the risk for any bugs
-caused by definition inconsistency.
-
 .. tip::
 
-    If the application uses full language + territory locales (e.g. ``fr_FR``,
-    ``fr_BE``), you can use the language part only in your routes (e.g. ``fr``).
-    This prevents having to define multiple paths when you want to use the same
-    route path for locales that share the same language.
+    HTML forms only support ``GET`` and ``POST`` methods. If you're calling a
+    route with a different method from an HTML form, add a hidden field called
+    ``_method`` with the method to use (e.g. ``<input type="hidden" name="_method" value="PUT" />``).
+    If you use :doc:`Symfony forms </forms>` this is done automatically for you.
```

Comment:
```suggestion
    If you use :doc:`Symfony Forms </forms>` this is done automatically for you.
```

very minor, but I think we always capitialize package names

Reply:
Ryan also complained ... I'm going to reword this.